tests/unit/test_property.py

```diff
@@ -8,3 +8,9 @@ def setup(self):
         self.object = PropertyModel()
         self.custom_404_msg = "Property not found"
         self.schema = PropertySchema
+
+    def test_tenants_attached_to_property(self, create_lease):
```

Comment:
When we are using the `create_{db_entry}` fixtures for tests, we should be using the test databases from now on.  
You can do this a couple of ways. You can add `empty_test_db` as the second parameter (after `self`), or you can add the decorator line 
```python
@pytest.mark.usefixtures("empty_test_db")
```
above the class definition. 

I think my preferred method is to use the decorator (since it will apply to all methods within the test class, in case people forget), but either way is valid.

Add this in, and I think we should be ready to merge this. 😄

Reply:
This test actually uses the empty test db through inheritance; otherwise the test would have failed without a db being specified.

Reply:
Oh, you're right! Totally missed that!